src/rdgnu/.gitignore

```diff
@@ -0,0 +1,34 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
```

Comment:
Let's merge this with the root gitignore. Also, let's rename this directory to just "frontend"